packages/relay/tests/lib/eth/eth-helpers.ts

```diff
@@ -25,15 +25,52 @@ import constants from '../../../src/lib/constants';
 import HbarLimit from '../../../src/lib/hbarlimiter';
 import HAPIService from '../../../src/lib/services/hapiService/hapiService';
 import MockAdapter from 'axios-mock-adapter';
-import { MirrorNodeClient } from '../../../src/lib/clients/mirrorNodeClient';
+import { MirrorNodeClient } from '../../../src/lib/clients';
 import { EthImpl } from '../../../src/lib/eth';
+import EventEmitter from 'events';
 
-export function contractResultsByNumberByIndexURL(number: number, index: number): string {
-  return `contracts/results?block.number=${number}&transaction.index=${index}&limit=100&order=asc`;
+export function getQueryParams(params: object) {
```

Comment:
Doesn't this function already exist in `helpers.ts`?
Could it be reused?

Reply:
Yes, it could be reused but it needs to be extended to also support arrays for values in the `params` object, I will extend it and remove this one. Thanks!